.codecov.yml

```diff
@@ -0,0 +1,19 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+coverage:
+  precision: 1
+  round: nearest
+  range: '10...100'
```

Comment:
this just configures the color threshholds

Reply:
Maybe throw a YAML comment in there if there's details that will help the next time we're in here making changes?

Reply:
f238bfd